frontend/src/components/MapView/index.tsx

```diff
@@ -517,6 +518,7 @@ const MapView = memo(({ classes }: MapViewProps) => {
             </Grid>
             {renderedGridItemAlertForm}
           </Grid>
+          <DataViewer />
```

Comment:
Let's add a comment noting that at the moment, this is only used for Cambodia. In the future, we could also investigate moving this into the chart drawer.